app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
+use App\User;
+use Illuminate\Support\Facades\Auth;
+use Validator;
+use App\Http\Requests\EditProfile;
+use Illuminate\Support\Facades\Hash;
+
+class UserController extends Controller
+{
+    public function edit($id)
+    {
+        $user = User::find($id);
+        return view('profile_edit', compact('user'));
+    }
+    public function update(EditProfile $request, $id)
```

Comment:
Please put a new line in between functions. It's helps in making your code readable.

Reply:
I edited it.